infra/experimental/README.md

```diff
@@ -0,0 +1,21 @@
+# Shell Injection Detection with `ptrace`
+
```

Comment:
Can we add a quick intro section with some details on how this works?

i.e. the ptrace hooks into clone/execve and through fuzzing dictionaries we try to run our "tripwire" program to detect shell injection.